src/wheel.js

```diff
@@ -4,8 +4,7 @@ class Wheel{
 		this.currentWheelValue = [];
 	}
 	SetWheelValue(){
-		let randomValue = this.wheelValue
-		
+		let randomValue = this.wheelValue.slice()
```

Comment:
Everything else looks good but this one. Can you please explain it to me when we meet? This is interesting and I would like to learn more about it